viite-UI/src/model/ProjectCollection.js

```diff
@@ -117,6 +116,8 @@
           publishable: result.publishable
         };
         projectErrors = result.projectErrors;
```

Comment:
we can start to use the existing methods setProjectErrors, setReservedParts and setFormedParts